src/character.cpp

```diff
@@ -2319,6 +2324,26 @@ void Character::mod_int_bonus( int nint )
     int_cur = std::max( 0, int_max + int_bonus );
 }
 
+namespace io
+{
+template<>
+std::string enum_to_string<Character::stat>( Character::stat data )
```

Comment:
`enum_to_string` is intended for use with serialization.  We shouldn't be using translations in here.  `load_bionic` will fail in other languages.